include/spirv-tools/libspirv.h

```diff
@@ -54,6 +54,7 @@ typedef enum spv_result_t {
   SPV_ERROR_INVALID_LAYOUT = -12,
   SPV_ERROR_INVALID_CAPABILITY = -13,
   SPV_ERROR_INVALID_DATA = -14,  // Indicates data rules validation failure.
+  SPV_ERROR_RESERVED_OPCODE = -15,
```

Comment:
Sounds good.  INVALID_VALUE sounded to me like it would be used in cases where a constant was initialized to the wrong type or something.

Updated to use INVALID_VALUE instead.